docs/management/admin/ts-management.asciidoc

```diff
@@ -142,4 +142,58 @@ To debug and resolve the issue, follow these steps:
 ** `ping -4 -c 1 localhost` (Linux)
 ** `ping -c 1 localhost` (macOS)
 ** `ping -4 localhost` (Windows)
+====
+
+[discrete]
+[[defend-deployment]]
+.{elastic-defend} deployment issues
+[%collapsible]
+====
+
+{elastic-defend} deployments may result in warnings or errors in the endpoint **Policy status** in {fleet} due to incorrect mobile device management (MDM) configuration, or certain permissions not being enabled for {elastic-endpoint}. Here are some common issues that can cause the policy status to result in a warning or failure.
+
+[float]
+==== Connect Kernel has failed
```

Comment:
Is this an actual warning or failure message that gets returned? If so, it might be worth clarifying in the intro for this section.

Reply:
It's not a verbatim error message—there would be a `Failed` status next to the **Connect Kernel** line in the UI.